plugins/print/print_main.ml

```diff
@@ -45,6 +45,11 @@ let sec_name memory fn sub =
   | None -> "bap.virtual"
   | Some name -> name
 
+let print_spec ppf proj =
+  match Project.get proj Image.specification with
+  | None -> Format.fprintf ppf "Image specification not found\n"
```

Comment:
Don't print errors into the standard output. People will be parsing it so everything you're printing here should be in the specified format, otherwise it is a bug on your side, that will make people very angry.Use stderr for errors and diagnostic messages.  In our case, it is hardly an error, maybe not even a warning. So just print nothing, and issue some dignostic in the log, using `info/warning` functions. Its your call.